Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Cultural Naming Improvements (Elven Group)

## Changelog:
- Significantly increased library character and dynastic names for High Elves, Blood Elves, Night Elves, Highborne, Nightborne, Satyr and Naga using NPC names and lore friendly derived naming conventions as a guide.
  - Removed generated first names for Blood Elves and High Elves. 
  - Reduced generated first names for Night Elves and Highborne by one half.
  - Reduced generated dynasty names for Night Elves by half.
  - Replaced generated Highborne dynasty names with selected Kaldorei placeholder dynasty names.
  - Introduced some cross-cultural dynastic names where appropriate.


## Developer changelog:

- Added WoW/RTS NPC names to High Elves, Blood Elves, Night Elves, Nighborne and Naga.
- Dynastic names with `"dynn_x"` point to historical dynasties described in `common/dynasties`. Additions without `"dynn_x"` are lore sourced but not described in 'common/dynasties`.
- Updated formatting in-line with #820 ; categories ordered by lore, lore-inspired (including from other subcultures), random generated.
- Pointed blood elf "dynn_x" defined in common\dynasties to blood elf culture and removed incorrect "dynn_" prefixes for high elven dynasties.
- Added missing dynn_Hawkspear, Brightsun, Salonar from bottom of Commonn\Dynasties to Blood Elf culture
- Replaced duplicate random dynasty names and thematically ineffective random dynasty names with lore names 
  - (Changelog incomplete; check changed files)

<details>
<summary> Click to show </summary>

Blackwing -> Dawnbreeze
Rosestrider -> Brightsong
Firebender -> Snowdawn
Phoenixwalker -> Daybreak
Bloodwood -> Bloodwrath
Dayreaver -> Daybreaker
Dayspear -> Dayblaze
Emberseeker -> Emberfall
Silvergaze -> Silversun
Stargaze -> Starsorrow

Flameflame -> Flameheart
Dewblade -> Dawnfury
Dewbringer -> Dawnstrike
Dewspear -> Dawnglaive
Heartheart -> Heartshade
Redseeker -> Redmorn (Blood Elf)
Redseeker -> Rosebreeze (High Elf)
Blackspell -> Brighthand (High Elf)
Blackspell -> Bloodblade (Blood Elf)
Blackfeather -> Bloodfire
Darkfury -> Dawnshield
Heartsprinter -> Morningsong
Coldbringer -> Ebonscorch
Coldheart -> Emberbinder
Coldshadow -> Brightstar
Downfall -> Sunhollow
Redbringer -> Riverwind (Blood Elf)
Redbringer -> Riverwind (High Elf)
Redforge -> Solarforge
Phoenixgaze -> Sunborn
Phoenixdown -> Truehair
Nightsworn -> Nightscorn
Colddown -> Cinderweave
Downflare -> Darkflare
Mirthsprinter -> Morrowsong
Mirthbinder -> Magehand
Downblade -> Everspring
Blackforge -> Lighthammer
Dewspell -> Dewspring
Downgaze -> Dawnrunner
Of the Nightfall -> Netherburn
Mirthflare -> Mourningsun
Darkdown -> Darkleaf
Autumnshade -> Springrunner
Coldwood -> Autumnwood
Morningbringer -> Morningstar
Moonbringer -> Moonvale (Blood Elf)
Moonbringer -> Loresong (High Elf)
Leafdown -> Feltouch
Leafblossom -> Dawnrose
Lightwood -> Lightblade
Heartwhisper -> Heartspear
Redbrook -> Roseblade
Leaflight -> Lightfall
Coldsworn -> Sunshadow
Nightsky -> Sunlance (High Elf)
Nightsky -> Azuresong (Blood Elf)
Morningshadow -> Sunshroud
Nightvale -> Belore

</details>


## Tests:
   - For all newly added first names, this error is returned:


# How to test:
- Check if character and dynasty names from lore and lore-inspired categories appear when generating cultural names in the ruler designer. 
- Ensure no unintended text such as slashes or prefixes like "dynn_x" show up. 
- Review added character and dynasty names for suggestions especially regarding enduring character/dynasty names across subcultures.
